dev-python/spyder/spyder-4.0.1.ebuild

```diff
@@ -1,67 +1,64 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Don't use `git mv` on this, but rather make one commit for a whole new file addition, then one new commit for removing 4.0.0_beta1.

Reply:
Fixed, should I remove the other old versions of spyder and spyder-kernels as well? Because some of those versions are quite old.